src/Build/Resources/Strings.resx

```diff
@@ -331,6 +331,18 @@
   </data>
   <data name="TryingExtensionsPath" xml:space="preserve">
     <value>Trying to import {0} using extensions path {1}</value>
+  </data>
+    <data name="OverrideUsingTaskElementCreated" xml:space="preserve">
```

Comment:
Why do all of these need error codes? I would've thought just the DuplicateOverrideUsingTaskElement should.